### PR TITLE
Undislocation Tweaks

### PR DESCRIPTION
**Pull request by Geevies**
* You can now undislocate joints even while stunned and weakened, but not unconscious.
* Added pain to undislocating joints. I suggest giving your patients painkillers, medical.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1661,6 +1656,7 @@
 		U.visible_message("<span class='danger'>[U] pops [S]'s [current_limb.joint] back in!</span>", \
 		"<span class='danger'>You pop [S]'s [current_limb.joint] back in!</span>")
 	current_limb.undislocate()
+	custom_pain("A sharp pain jolts through your body as your [current_limb.joint] is reset!", 15)
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/human.dm]: Meeeh. I'd pump this pain up, honestly.

File: code/_helpers/unsorted.dm [hunk 2]
```diff
@@ -678,7 +678,7 @@ proc/GaussRandRound(var/sigma,var/roundto)
 	if (progbar)
 		qdel(progbar)
 
-/proc/do_after(mob/user as mob, delay as num, needhand = TRUE, atom/movable/act_target = null, use_user_turf = FALSE, display_progress = TRUE, datum/callback/extra_checks)
```
Comment (NonQueueingMatt) [inline code/_helpers/unsorted.dm]: Delay as num is actually a proper usage of this, though.

Comment (skull132) [reply code/_helpers/unsorted.dm]: No it isn't. `as X` syntax only applies to verbs and procs-as-verbs.

File: code/modules/mob/living/carbon/human/human.dm [hunk 3]
```diff
@@ -1611,24 +1611,19 @@
...
-	if(usr.restrained())
-		to_chat(usr, "You are restrained and cannot do that!")
+	if(!usr.canClick())
+		to_chat(usr, span("warning", "You cannot do this so soon!"))
 		return
+	usr.setClickCooldown(20)
 
-	var/mob/S = src
+	var/mob/living/carbon/human/S = src
 	var/mob/U = usr
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/human.dm]: Both of these vars are pointless. Remove.

**Pull request closed without merging**